doc/install/upgrading-ceph.rst

```diff
@@ -106,7 +106,7 @@ To upgrade monitors, perform the following steps:
 
 #. Restart each monitor. For Ubuntu distributions, use::
 
-	sudo restart ceph-mon id={hostname}
+	sudo systemctl restart ceph-mon@{hostname}.service
```

Comment:
could you note in the commit message that 

> before this change, we were using the "restart" command which is a symlink to initctl(8). and it's offered by upstart. since ubuntu/vivid, ubuntu moved from upstart to systemd. see https://lists.ubuntu.com/archives/ubuntu-devel/2016-July/039465.html

to help the reviewers and posterities to get more context of this change?

Reply:
Done.